src/components/views/dialogs/AnalyticsLearnMoreDialog.tsx

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2021 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import BaseDialog from "./BaseDialog";
+import { _t } from "../../../languageHandler";
+import DialogButtons from "../elements/DialogButtons";
+import React from "react";
+import Modal from "../../../Modal";
+import SdkConfig from "../../../SdkConfig";
+
+export enum ButtonClicked {
+    Primary,
+    Cancel,
+}
+
+interface IProps {
+    onFinished?(buttonClicked?: ButtonClicked): void;
+    analyticsOwner: string;
+    privacyPolicyUrl?: string;
+    primaryButton?: string;
+    cancelButton?: string;
+    hasCancel?: boolean;
+}
+
+const AnalyticsLearnMoreDialog: React.FC<IProps> = ({
```

Comment:
This dialog looks eerily similar to the new Compound dialog from design. While this PR has been open, a new base class `ScrollableBaseModal` was introduced which might fit the design (at least, it does from what I can see at a glance).

I believe we're supposed to be using Compound for everything these days, so it'd be good to either align the dialog with that or fix the Compound design to match the proposal here.

Reply:
@nadonomy - should this dialog be using Compound?

@turt2live - Where do I go to view the dialog in Compound which you're referring to?

Reply:
I never know where to find it, but polls use a new kind of dialog that is supposed to (eventually) be used everywhere, aiui.

Reply:
> @nadonomy - should this dialog be using Compound?

We haven't set a formal mandate for new feature development to use new components yet.

If it's trivial to swap out now with the class Travis mentioned then sgtm and we save some debt/cleanup later, but wouldn't block on it.

Reply:
It's not a huge amount of work but non-trivial - also I think Travis is suggesting the new dialog looks slightly different..?

Reply:
There might be a subtle visual difference, comparing the normal to scrollable versions here:

https://www.figma.com/file/X4XTH9iS2KGJ2wFKDqkyed/Compound?node-id=643%3A101

But I think we should just move forward on this PR:

1. This modal isn't scrollable.
2. I think we actually just want one one true modal component which _can_ scroll when needed.
3. In future we plan to standardise on re-usable components. If this component isn't documented well enough (Storybook? Document somewhere detailing components and use? Better notes in Figma? etc) I think we're worrying prematurely.

So would just prefer to stick the course on analytics and avoid this as it's not immediately impactful and as you said non-trivial.

Reply:
hmm, as you say: the dialog is different enough to be, well, different. Probably not worth worrying about here, I guess.

It is more debt we have to worry about in the future, but it doesn't sound like there's a firm requirement to use the new dialog structure yet, so would rather steer away from accidentally implementing a 5th dialog style that may or may not get used.